### PR TITLE
#16 - code style violations in headers

### PR DESCRIPTION
**Pull request by maurezen**
Added license.txt to configuration. Please also note jcabi/jcabi-parent#19

### CODE REVIEW

**Comment by dmarkov** [comment]
Many thanks for the PR, let me find a reviewer for it

**Comment by maurezen** [comment]
@dmarkov any progress with that?

**Comment by maurezen** [comment]
@dmarkov @yegor256 please find someone to review this PR. It's been quite a while.

**Comment by dmarkov** [comment]
@carlosmiranda please review this one

File: pom.xml [hunk 1]
```diff
@@ -99,4 +99,27 @@
             </plugin>
         </plugins>
     </build>
+    <!--
+        @todo #16 Remove this declaration when parent pom would 
+        fix issue jcabi/jcabi-parent#19
+         -->
```
Comment (carlosmiranda) [inline pom.xml]: Let's align this line with line 102 above.

**Comment by carlosmiranda** [comment]
@maurezen One minor comment.

**Comment by maurezen** [comment]
@carlosmiranda fixed.

**Comment by carlosmiranda** [comment]
@rultor can we merge?

**Comment by yegor256** [comment]
@rultor I fixed config, try to merge again

**Comment by yegor256** [comment]
@rultor merge it! :)

File: pom.xml [hunk 2]
```diff
@@ -99,4 +99,27 @@
             </plugin>
         </plugins>
     </build>
+    <!--
+        @todo #16 Remove this declaration when parent pom would 
+        fix issue jcabi/jcabi-parent#19
```
Comment (carlosmiranda) [inline pom.xml]: Let's add a single space here (`f` in line 104 should be aligned with `t`, not `@`).

**Comment by carlosmiranda** [comment]
@maurezen rultor is complaining about puzzle format. Let's fix it (see my latest code comment above)

**Comment by maurezen** [comment]
@carlosmiranda done.

**Comment by carlosmiranda** [comment]
@rultor Try the merge again please

**Comment by yegor256** [comment]
@rultor merge it

**Comment by dmarkov** [comment]
@carlosmiranda done, I added **20 mins** in payment `50570821`

added +20 to your rating, now it is equal to [+1750](http://www.netbout.com/b/35023?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy pls

**Pull request merged**